app/templates/sites/companies/company/edit.hbs

```diff
@@ -0,0 +1,14 @@
+<h2>Edit Company</h2>
+
+<div class='form-group'>
+  <label for='company-name'>Company Name</label>
+  <br>
+  {{input
+      value=name
+      size=50
+      placeholder='Company, got a name?'
+      id='company-name'
```

Comment:
Do you need an `id` here? Could a classname or data-test-selector accomplish this?